test/parallel/test-intl-no-icu-data.js

```diff
@@ -0,0 +1,10 @@
+// Flags: --icu-data-dir=test/fixtures/empty/
```

Comment:
I guess you're right but see https://github.com/nodejs/build/issues/419.  Since we don't test non-intl builds I feel somewhat ambivalent about complicating the test.  I'll update it if you feel strongly about it.